custom/ewsghana/tasks.py

```diff
@@ -59,7 +59,8 @@ def migration_task():
 
 
 # Alert when facilities have not been reported continuously for 3 weeks
-@periodic_task(run_every=crontab(hour=10, minute=00),
+# TODO change to trigger everyday
+@periodic_task(run_every=crontab(day_of_week="1-6", hour=10, minute=00),
```

Comment:
not sure if i understand - is this supposed to be a weekly alert, or a daily alert?